2021-11-09 15:48:14,283 INFO    MainThread:16792 [wandb_setup.py:_flush():69] setting env: {}
2021-11-09 15:48:14,283 INFO    MainThread:16792 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-09 15:48:14,283 INFO    MainThread:16792 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_154814-2i3u33r1\logs\debug.log
2021-11-09 15:48:14,283 INFO    MainThread:16792 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_154814-2i3u33r1\logs\debug-internal.log
2021-11-09 15:48:14,284 INFO    MainThread:16792 [wandb_init.py:init():375] calling init triggers
2021-11-09 15:48:14,284 INFO    MainThread:16792 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-11-09 15:48:14,284 INFO    MainThread:16792 [wandb_init.py:init():424] starting backend
2021-11-09 15:48:14,284 INFO    MainThread:16792 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-11-09 15:48:14,288 INFO    MainThread:16792 [backend.py:ensure_launched():135] starting backend process...
2021-11-09 15:48:14,381 INFO    MainThread:16792 [backend.py:ensure_launched():139] started backend process with pid: 26252
2021-11-09 15:48:14,381 INFO    MainThread:16792 [wandb_init.py:init():429] backend started and connected
2021-11-09 15:48:14,382 INFO    MainThread:16792 [wandb_init.py:init():477] updated telemetry
2021-11-09 15:48:14,382 INFO    MainThread:16792 [wandb_init.py:init():500] communicating current version
2021-11-09 15:48:15,709 INFO    MainThread:16792 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-09 15:48:15,710 INFO    MainThread:16792 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-11-09 15:48:15,957 INFO    MainThread:16792 [wandb_init.py:init():540] starting run threads in backend
2021-11-09 15:48:20,492 INFO    MainThread:16792 [wandb_run.py:_console_start():1601] atexit reg
2021-11-09 15:48:20,493 INFO    MainThread:16792 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-11-09 15:48:20,493 INFO    MainThread:16792 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-11-09 15:48:20,495 INFO    MainThread:16792 [wandb_run.py:_redirect():1536] Redirects installed.
2021-11-09 15:48:20,495 INFO    MainThread:16792 [wandb_init.py:init():565] run started, returning control to user process
2021-11-09 15:48:20,495 INFO    MainThread:16792 [wandb_config.py:__setitem__():141] config set data = {'name': 'bike'} - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000026599ADD8E0>>
2021-11-09 15:48:20,495 INFO    MainThread:16792 [wandb_run.py:_config_callback():843] config_cb data {'name': 'bike'} None
2021-11-09 15:48:45,107 INFO    MainThread:16792 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-11-09 15:48:45,107 INFO    MainThread:16792 [wandb_run.py:_restore():1543] restore
2021-11-09 15:48:45,440 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 945
}

2021-11-09 15:48:45,547 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 945
}

2021-11-09 15:48:45,678 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 945
}

2021-11-09 15:48:46,334 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 13658
}

2021-11-09 15:48:46,445 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 945
  total_bytes: 13974
}

2021-11-09 15:48:46,555 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:46,666 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:46,777 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:46,885 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:46,997 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:47,109 INFO    MainThread:16792 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13974
  total_bytes: 13974
}

2021-11-09 15:48:48,242 INFO    MainThread:16792 [wandb_run.py:_show_summary():1848] rendering summary
2021-11-09 15:48:48,246 INFO    MainThread:16792 [wandb_run.py:_show_files():1915] logging synced files
